realsense2_camera/tools/frame_latency/include/frame_latency.h

```diff
@@ -0,0 +1,29 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved
+
+#pragma once
+
+#include <rclcpp/rclcpp.hpp>
+#include "sensor_msgs/msg/image.hpp"
+
+namespace rs2_ros
+{
```

Comment:
You don't have clang-format on there? :)

Reply:
I will figure something out :)

Reply:
Done